alexandrie/src/config/mod.rs

```diff
@@ -30,12 +30,19 @@ pub struct GeneralConfig {
     /// The port to listen on.
     pub port: u16,
 }
-
+/// The asset confiuration options struct
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct AssetCoonfig {
+    /// asset dir path
+    pub path: String,
+}
 /// The application configuration struct.
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub struct Config {
     /// General instance configuration options.
     pub general: GeneralConfig,
+    /// asset configuration options
+    pub asset: AssetCoonfig,
```

Comment:
Just extending my suggestion to add an `s` to `asset`.
```suggestion
    /// Asset configuration options
    pub assets: AssetsConfig,
```

Reply:
that is ok